resource/ui/ts/models/timeseries.ts

```diff
@@ -7,6 +7,8 @@
  */
```

Comment:
It should be named metrics, i will rename it in a subsequent PR to avoid difficult diffs.